Seminar_5/Task_26.py

```diff
@@ -0,0 +1,14 @@
+# Задача 26. Напишите программу, которая принимает на вход два числа А и В,
+# и возводит число А в квадрат В с помощью рекурсии.
+a = int(input('Введите число А: '))
+b = int(input('Введите число В: '))
+
+
+def square(a, b):
+    if b == 1:
+        return a
+    else:
+        return a * square(a, b-1)
```

Comment:
Тут конструкция else не нужна
```suggestion

    return a * square(a, b-1)
```